pkg/tests/apis/folder/folders_test.go

```diff
@@ -1168,3 +1168,84 @@ func TestIntegrationFoldersGetAPIEndpointK8S(t *testing.T) {
 		}
 	}
 }
+
+// Reproduces a bug where folder deletion does not check for attached library panels.
+func TestFolderDeletionBlockedByLibraryElements(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
```

Comment:
This test is NOT an integration test, because it doesn't have the right name. I'd suggest to use `tests.SkipIntegrationTestInShortMode` method which checks for this as well.